### PR TITLE
completed setup

### PR DESCRIPTION
**Pull request by AFerg15**

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class TasksController < ApplicationController
+
+    def index
+        tasks = Task.all
+        render json: jobs
```
Comment (sjproctor) [inline app/controllers/tasks_controller.rb]: Looks like your variable is named tasks but your are rendering jobs

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, NavItem } from 'reactstrap';
```
Comment (sjproctor) [inline app/javascript/components/App.js]: Are you using these in this component?

**Review by sjproctor — APPROVED** [review]
Looks good, just keep an eye on where you copy and paste from Cat Tinder. Make sure you are getting everything changed.

File: app/controllers/tasks_controller.rb (hunk 1 above)
Comment (AFerg15) [reply app/controllers/tasks_controller.rb]: smh...thank you :) I fixed before merging.

**Pull request merged**